test/CommentTester.cs

```diff
@@ -100,18 +100,22 @@ FOR INSERT
 ";
 
         private readonly string AutoGenerateComment = String.Format(
```

Comment:
This was one of the concerns I had and have discussed with @Tony-Zheng-Zocdoc extensively. The reason of not having a centralized variable is so it is easier to catch errors when they diverge, but I also see your point.